hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneClientAdapterImpl.java

```diff
@@ -823,8 +824,10 @@ private List<FileStatusAdapter> listStatusBucketSnapshot(
 
     while (snapshotIter.hasNext()) {
       OzoneSnapshot ozoneSnapshot = snapshotIter.next();
-      res.add(getFileStatusAdapterForBucketSnapshot(
-          ozoneBucket, ozoneSnapshot, uri, owner, group));
+      if (SnapshotInfo.SnapshotStatus.SNAPSHOT_ACTIVE.name().equalsIgnoreCase(ozoneSnapshot.getSnapshotStatus())) {
```

Comment:
do we need ignoreCase here or can we skip it?

Reply:
I tried it. It also works without ignoreCase.

Reply:
I did to be more cautious. I removed it.